### DISCUSSION TITLE
How to properly map entities fields during APIv2 migration

### DISCUSSION THREAD

**Discussion by Zanfo96** [fattureincloud/api#217] [Q&A] [answered]
Hello,

I am currently working on APIv2 migration. When handling the mapping of FIC APIv2 responses, some doubt raises about old entities fields and new entities fields and how to mirror those two mappings in order to match them. 

I have this table which exposes the doubts and the mapping already understood:

![image](https://user-images.githubusercontent.com/82850061/211199063-325134ab-6fd0-4b10-8ea3-509e03861666.png)
![image](https://user-images.githubusercontent.com/82850061/211199076-9fafe085-f9cc-4e7a-93d4-9725aac58ee6.png)

Maybe is there some kind of specific documentation I can check out?

Thanks you very much and have a nice sunday!

**Comment by emajo** [comment, marked as answer]
Hi @Zanfo96 your mappings seem to be correct, `dettagli_documento.numero_fattura` is actually `receivedDocument.invoice_number` and `lista_articoli.ordine` is not present in the API V2, to be sure of the mappings I suggest to create a document with the API v1 and make a GET request of it with the API V2, this should solve all your doubts.

Thanks,
Emanuele.

**Reply by valmoz** [reply]
Hi @Zanfo96 did we provide you with enough information to support you appropriately? Do you have further questions?
